src/org/sosy_lab/java_smt/basicimpl/AbstractFormulaManager.java

```diff
@@ -135,7 +138,8 @@ protected AbstractFormulaManager(
       @Nullable
           AbstractQuantifiedFormulaManager<TFormulaInfo, TType, TEnv, TFuncDecl> quantifiedManager,
       @Nullable AbstractArrayFormulaManager<TFormulaInfo, TType, TEnv, TFuncDecl> arrayManager,
-      @Nullable AbstractSLFormulaManager<TFormulaInfo, TType, TEnv, TFuncDecl> slManager) {
+      @Nullable AbstractSLFormulaManager<TFormulaInfo, TType, TEnv, TFuncDecl> slManager,
+      @Nullable StringFormulaManager strManager) {
```

Comment:
This looks strange: All managers are "Abstract*Managers" and the new StringFormulaManager is not.
This might be an effect of the small number of solvers supporting String theory.
For future extensions, this could be changed.